docs/user/README.md

```diff
@@ -1,44 +1,67 @@
-# Glean SDK
+# Introduction
 
-![Glean logo](glean.jpeg)
+Glean is a modern approach for a Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+
+## Contact
 
-The `Glean SDK` is a modern approach for a Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+To contact the Glean team you can:
 
-To contact us you can:
 - Find us in the [#glean channel on chat.mozilla.org](https://chat.mozilla.org/#/room/#glean:mozilla.org).
-- To report issues or request changes, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean%3A+SDK&priority=P3&status_whiteboard=%5Btelemetry%3Aglean-rs%3Am%3F%5D).
 - Send an email to *glean-team@mozilla.com*.
 - The Glean SDK team is: *:janerik*, *:dexter*, *:travis*, *:mdroettboom*, *:gfritzsche*, *:chutten*, *:brizental*.
```

Comment:
Should we update this? Georg's out for a while now :(

Reply:
Or we can move his name to a "contributors" or "emeritus" section...

Reply:
"alumni" maybe? 

That would not fit in the `# Contact` section though, IMO. How about we remove his name here and file a bug for adding a `contributors` section somewhere else? Maybe the Github README? We could maybe even automate that based on commit history or something. Outside contributors might appreciate that :D

Reply:
Let's leave it and tackle that separately.